packages/synapse-react-client/src/components/TimelinePlot/TimelinePhase.tsx

```diff
@@ -201,11 +203,11 @@ const TimelinePhase = ({
   schema,
   widthPx,
 }: TimelinePhaseProps) => {
+  const observationsSql = 'SELECT * FROM syn51735464'
```

Comment:
This should not be hard-coded, it should be a prop that ultimately comes from `observationsSql` in the NF portal resources.ts